File: Comparacao/Go/operations/operations.go
package main

import "fmt"

func main() {
	x,y := 5,3
	
	fmt.Println("x + y = ", x+y)
	fmt.Println("x - y = ", x-y)
	fmt.Println("x * y = ", x*y)
	fmt.Println("x / y = ", x/y)
	fmt.Println("x mod y = ", x%y)

	varbool := true
	otherbool := false


  // O printf

	// Equivale ao printf do C
	
	fmt.Printf("Printf: \n")
  fmt.Printf("x + y = %d \n", x+y)

	// Equivalente da declaração e definição anterior
	//		var varbool bool = true
	//		var otherbool bool = false
	

  	/*
  	  Aliás, os comentarios são iguais ao C.
  	*/

	fmt.Println(varbool && otherbool)
	fmt.Println(varbool || otherbool)
	fmt.Println(!(varbool) == otherbool)
}

File: Comparacao/C/operations/operations.c
#include <stdio.h>
#include <stdbool.h>

int main() {
	int x = 5;
	int y = 3;

	printf("x + y = %d\n", x+y);
	printf("x - y = %d\n", x-y);
	printf("x * y = %d\n", x*y);
	printf("x / y = %d\n", x/y);
	printf("x mod y = %d\n", x%y);

	bool varbool = true;
	bool otherbool = false;

	// C é incapaz de imprimir tipos booleanos
	printf(varbool && otherbool? "true" : "false");
	printf("\n");
	printf(varbool || otherbool? "true" : "false");
	printf("\n");
	printf(!(varbool) == otherbool?"true" : "false");
	printf("\n");

	return 0;
}
